src/Interactive/Host/Interactive/Core/InteractiveHostOptions.cs

```diff
@@ -24,10 +24,15 @@ internal sealed class InteractiveHostOptions
         public string? InitializationFilePath { get; }
 
         /// <summary>
-        /// Host culture used for localization of doc comments, errors.
+        /// Host culture used for data formatting.
         /// </summary>
         public CultureInfo Culture { get; }
 
+        /// <summary>
+        /// Host culture used for localization of doc comments, errors.
+        /// </summary>
+        public CultureInfo UICulture { get; }
```

Comment:
fwiw i don't know why we'd format data one way, but doc comments anotehr way.  BUt i also don't know anything about loc :)